src/test/groovy/org/shipkit/internal/gradle/init/tasks/InitShipkitFileTest.groovy

```diff
@@ -0,0 +1,70 @@
+package org.shipkit.internal.gradle.init.tasks
+
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import org.shipkit.internal.notes.vcs.GitOriginRepoProvider
+import spock.lang.Specification
+
+class InitShipkitFileTest extends Specification {
+
+    @Rule TemporaryFolder tmp = new TemporaryFolder()
+
+    def "does not modify shipkit file if it already exists"() {
+        given:
+        def shipkitFile = tmp.newFile()
+        shipkitFile << "foo"
+
+        when:
+        InitShipkitFile.initShipkitFile(shipkitFile, tmp.root)
+
+        then:
+        shipkitFile.text == "foo"
+    }
+
+    def "uses fallback repo if git call fails"() {
+        def shipkitFile = tmp.newFile()
+        def repoProvider = Mock(GitOriginRepoProvider)
```

Comment:
I personally prefer to have all variables defined with their formal types. It usually easier to read and understand. Your syntax was preferred in up to Spock 1.0 as in the one suggested by me there was a type warning in Idea. However, I tune it up in Spock 1.1 and it works fine with Idea, still be the same short (just type is where is should to be :) )